py/torch_tensorrt/dynamo/backend/backends.py

```diff
@@ -72,12 +82,12 @@ def fx_dynamo_backend(
         return gm.forward
 
 
-def compile_module(
+def _compile_module(
```

Comment:
Added "_" to name, as the function is intended only for internal/helper use (requires a pre-traced FX GraphModule)